### PR TITLE
[API Shield] Transform Rules fields

### PR DESCRIPTION
**Pull request by patriciasantaana**
### Summary

Add available Transform Rules fields to JWT Validation docs.

### Documentation checklist

- [ ] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.

### CODE REVIEW

File: src/content/docs/api-shield/security/jwt-validation/transform-rules.mdx [hunk 1]
```diff
@@ -35,3 +35,28 @@ As an example, to send the header `x-send-jwt-claim-user` request header to the
...
+- `http.request.jwt.claims.iss.values,`
+- `http.request.jwt.claims.jti,`
+- `http.request.jwt.claims.jti.names,`
+- `http.request.jwt.claims.jti.values,`
+- `http.request.jwt.claims.nbf.sec,`
+- `http.request.jwt.claims.nbf.sec.names,`
+- `http.request.jwt.claims.nbf.sec.values,`
+- `http.request.jwt.claims.sub,`
+- `http.request.jwt.claims.sub.names,`
+- `http.request.jwt.claims.sub.values`
```
Comment (Maddy-Cloudflare) [inline src/content/docs/api-shield/security/jwt-validation/transform-rules.mdx]:
```suggestion
- `http.request.jwt.claims.sub.values`,
```

File: src/content/docs/api-shield/security/jwt-validation/transform-rules.mdx [hunk 2]
```diff
@@ -35,3 +35,28 @@ As an example, to send the header `x-send-jwt-claim-user` request header to the
...
+- `cf.api_gateway.auth_id_present`
```
Comment (Maddy-Cloudflare) [inline src/content/docs/api-shield/security/jwt-validation/transform-rules.mdx]:
```suggestion
- `cf.api_gateway.auth_id_present`,
```

Comment (Maddy-Cloudflare) [reply src/content/docs/api-shield/security/jwt-validation/transform-rules.mdx]: Adding the commas on all rules, except the last one.

**Review by Maddy-Cloudflare — APPROVED** [review]

File: src/content/docs/api-shield/security/jwt-validation/transform-rules.mdx (hunk 2 above)
Comment (pedrosousa) [reply src/content/docs/api-shield/security/jwt-validation/transform-rules.mdx]: Personally, I wouldn't add commas on any of the list items. 😅

**Pull request merged**